### ISSUE TITLE
Problem creating nodes in v0.9.2

### ISSUE THREAD

**Issue by mikedanese** [GoogleCloudPlatform/kubernetes#3953]
This worked in v0.8.1 but does not in v0.9.2
```
[ubuntu@pilot10.DEV: ~]# cat docker10.json
{

  "apiVersion": "v1beta1",
  "id": "docker10.dev",
  "kind": "Minion",
  "labels": {
    "name": "docker10-dev",
  },
  "resources": {
    "capacity": {
      "memory": 3221225472,
      "cpu": 10000
    }
  },
}
[ubuntu@pilot10.DEV: ~]# kubectl create -f docker10.json
F0129 21:52:06.536562   12619 create.go:78] minion "docker10.dev" is invalid: namespace: invalid value 'default'
```

I see [this](https://github.com/GoogleCloudPlatform/kubernetes/blob/v0.9.2/pkg/api/validation/validation.go#L599) so I try adding an empty namespace to the object:
```
[ubuntu@pilot10.DEV: ~]# cat docker10.json
{
  "namespace": "",
  "apiVersion": "v1beta1",
  "id": "docker10.dev",
  "kind": "Minion",
  "labels": {
    "name": "docker10-dev",
  }
  "resources": {
    "capacity": {
      "memory": 3221225472,
      "cpu": 10000
    }
  },
}
[ubuntu@pilot10.DEV: ~]# kubectl create -f docker10.json
F0129 22:10:53.754913   16093 create.go:78] minion "docker10.dev" is invalid: namespace: invalid value 'default'
[ubuntu@pilot10.DEV: ~]# kubectl --namespace='' create -f docker10.json
F0129 22:11:09.210060   16141 create.go:78] minion "docker10.dev" is invalid: namespace: invalid value 'default'
```
No luck.

**Comment by mikedanese** [comment]
Looks like [this](https://github.com/GoogleCloudPlatform/kubernetes/blob/v0.9.2/pkg/registry/minion/rest.go#L119) is missing on create.

**Comment by mikedanese** [comment]
This was fixed in master by [this](https://github.com/GoogleCloudPlatform/kubernetes/commit/a0356bca9624827ae9f7072f60f567e0eb4b2c51#diff-f19a7777a36882029db668a427f950feR63). v0.9.2 requires a node's Namespace to be `""` [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/v0.9.2/pkg/api/validation/validation.go#L599) (which is broken because kubectl was appending a namespace) but master requires all TypeMeta to have a Namespace which I believe has the side effect of fixing the validation error, but haven't tested master yet.

**Comment by mikedanese** [comment]
This https://github.com/mikedanese/kubernetes/commit/5b37a6b596eed35879314a993c18a95a79f7487d applied to the release-0.9 branch fixes the issue.

**Comment by dchen1107** [comment]
cc/ @brendandburns Looks like we need to cut another patch release.

**Comment by brendandburns** [comment]
roger, wilco.

**Comment by mikedanese** [comment]
Fixed in 0.9.3
